packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
+
+- When the date picker is clicked, it takes today's date.
+- It is not possible to write the date from the keyboard.
+- It is not possible to change the date format. However, the date format will be based on the browser locale. 
+- A Date picker can used simultaneously with a Time picker in a form.
+- The PopOver closes after selecting a date.
+- One single format of date is currently available.
```

Comment:
Technically, it's not true: an american person and a french person will see different formatting on the screen

Reply:
Does it depend on the browser locale?

Reply:
yup

Reply:
Okay, when I commit the changes, I'll add the detail then :D Thanks!